cmd/up/stignore.go

```diff
@@ -41,7 +42,10 @@ func addStignoreSecrets(dev *model.Dev) error {
 		}
 		infile, err := os.Open(stignorePath)
 		if err != nil {
-			return err
+			return o_error.UserError{
```

Comment:
I didn't provide a suggestion change for this and now it's broken. You'll want to test this after applying.

```suggestion
			return oktetoError.UserError{
```